scripts/dev/downloader.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+downloader() { 
+    if command -v curl 2>/dev/null; 
+    then 
+    	curl -LO --retry 20 -O --progress $1 $2 $3; 
+    else 
+        wget $1 $2 $3 2> /dev/null; fi; 
```

Comment:
You can pass all arguments using $@ instead of just passing the first 3.

But also i'm not sure we should pass any arguments at all like this since arguments to wget and curl are different so this won't work most of the times.

For example some of the scripts calling this use -q which for wget is the intended quiet so there's not so much output but for curl means "Disable .curlrc" which in this case is probably not a problem but in the case of other arguments might have unintended consequences.

I would just add the arguments we use by now, i think rn only -q, and parse them in the script then pass the corresponding arguments to wget or curl and fail if the arguments passed are not in the list of the ones we pass

Reply:
Yeah great idea... didn't know about $@. Awesome.